qa/TL1_custom_cpp_lib/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+test_body() {
+    mkdir -p /tmp/custom_cpp_lib
+    git clone https://github.com/NVIDIA/DALI /tmp/dali
```

Comment:
Why you need to clone whole DALI, are it suppose to test the wheel?
Again, use `mktemp` instead of hard code `/tmp/dali`.

Reply:
Also you are testing main branch, no the code you actually runs. We copy the DALI source code into our test container so how about using it if you really must.

Reply:
Changed to include the script from `/opt/dali/tools...`